counterexamples/transportation/connector/bad-geometry-type.json

```diff
@@ -9,6 +9,7 @@
     "theme": "transportation",
     "type": "connector",
     "version": 2,
-    "updateTime": "2023-02-23T00:04:00-08:00"
+    "updateTime": "2023-02-23T00:04:00-08:00",
+    "level": 3
```

Comment:
Can you please create a new counter-example for this instead of appending to this one?

---

This file is called `bad-geometry-type.json` and is meant to "test" the geometry field.

It would be preferable to create some kind of `level-property-not-allowed.json` for the level property.

---

Would you also mind using the `extExpectedErrors` feature on your new counterexample to specify the expected error output?

Reply:
Yup will separate the tests. How is extExpectedErrors triggered as it's not super clear since most tests don't use this feature?

Reply:
I recommend you `grep`† for other test files that use `extExpectedErrors`, and also look at how it is used in `test.sh`. Maybe experiment by adding it to your counterexample file and seeing what happens?

@marinrelatic did some dive-deep diagnosis on this feature and fixed some problems with it in https://github.com/OvertureMaps/schema/pull/107, so he can probably help explain how it works if you get stuck. If you're still stuck after, come to me!

---

† *By `grep` I mean use your favorite find-in-files search utility!

Reply:
resolved. made a separate test with correct `extExpectedErrors` string